tests/tests.lua

```diff
@@ -464,12 +437,11 @@ end
     it("should reject inspect of a bounty that waits IO for too long", function()
         local res = inspect_input(machine, {
             sender = HACKER1_WALLET,
-            opcode = CodecOpcodes.TestExploit,
```

Comment:
Inspect payloads don't have a `kind` field.
The back-end just assumes it's always a `TestExploit` object.